packages/flutter/test/rendering/proxy_box_test.dart

```diff
@@ -492,7 +493,7 @@ void main() {
   test('RenderMouseRegion can change properties when detached', () {
     renderer.initMouseTracker(MouseTracker(
       renderer.pointerRouter,
-      (_) => <MouseTrackerAnnotation>[],
+      (_) => (<MouseTrackerAnnotation, Matrix4>{} as LinkedHashMap<MouseTrackerAnnotation, Matrix4>),
```

Comment:
Why not just do this?:
```suggestion
      (_) => (LinkedHashMap<MouseTrackerAnnotation, Matrix4>()),
```

Reply:
Explained above